### PR TITLE
Consistent Theme Styling for Annotations + auto update on change 

### PR DESCRIPTION
**Pull request by JoeAtHPI**
Currently, many colors are hardcoded throughout the code. This leads to hardly readable, or unreadable text depending on the theme, e.g. the grey of the buttons in a dark theme
Screenshot 2023-05-16 193223
or backgrounds were inconsistent
Screenshot 2023-05-31 140808

With this MR, colors adjust themselves to the theme and change automatically when a new change is applied.
Many `applyUserInterfaceTheme` and UI constants were added. As the values morph for Probes was different with each class, fullfilling the same purpose, they were extracted into their own classes. `BPBrowser` now invalidates itself when a theme changed. This is because in the scenario
1) Browser display method `a` with probes
2) User switches to method `b`
3) User switches theme
4) User goes back to `a`
the probes in `a` would still have the old theme as they were not rendered before.

Below are two example themes.

Screenshot 2023-06-01 164550
Screenshot 2023-06-01 164915
Screenshot 2023-05-31 200753

### CODE REVIEW

File: .smalltalk.ston [hunk 1]
```diff
@@ -10,7 +10,7 @@ SmalltalkCISpec {
   #testing : {
     #categories : [ 'Babylonian-*' ],
     #coverage : {
-      #categories : [ 'Babylonian-Core', 'Babylonian-UI', 'Babylonian-Compiler', 'Babylonian-ProbeLog' ]
+      #categories : [ 'Babylonian-Core', 'Babylonian-UI', 'Babylonian-Compiler', 'Babylonian-Printbugger' ]
```
Comment (ekrebs5) [inline .smalltalk.ston]: This should stay ProbeLog

File: packages/Babylonian-UI.package/BPMethodExampleMorph.class/instance/addArgumentScriptMorphs.st [hunk 2]
```diff
@@ -6,7 +6,7 @@ addArgumentScriptMorphs
 		container := self newContainerMorph.
 		container cellPositioning: #topLeft.
 		labelMorph := ((argumentName , ': ') asMorph 
-			color: Color lightGray; 
+			color: self class textColor;
```
Comment (ekrebs5) [inline packages/Babylonian-UI.package/BPMethodExampleMorph.class/instance/addArgumentScriptMorphs.st]: This is in this method and #addReceiverConstructorMorph; maybe extract into its own method

**Review by ekrebs5 — CHANGES_REQUESTED** [review]
grafik

Looks good in general, just have a few inputs:
- it tries to readd the printbugger package; this is the probelog now
- see comment that one color could be extracted to its own function
- as seen in the screenshots, dark themes such as Monokai (dark) retain a light color for some Babylonian Morphs if the browser is open during the change; can this be fixed or will a browser reopen be neccessary here?

**Comment by JoeAtHPI** [comment]
@ekrebs5 did you mean the printbugger in the .ston? or elsewhere?

**Review by ekrebs5 — APPROVED** [review]

**Pull request merged**